uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-elasticsearch/src/main/java/org/uberfire/ext/metadata/backend/elastic/index/ElasticSearchIndexProvider.java

```diff
@@ -313,8 +313,16 @@ private String escape(String s) {
         return sb.toString();
     }
 
-    private List<String> indicesToLowerCase(List<String> indices) {
-        return indices.stream().map(String::toLowerCase).collect(Collectors.toList());
+    protected List<String> prepareIndices(List<String> indices) {
+        return indices.stream()
+                .map(index -> index.toLowerCase().replaceAll("/",
```

Comment:
You might want to call `.map(this::prepareIndex)` here just to concentrate all index sanitization logic in one place it needs to be extended in the future.